changelog.d/11849.misc

```diff
@@ -0,0 +1 @@
+Enable cache time-based expiry by default.
```

Comment:
It may be worth mentioning the new configuration options that are available, or otherwise pointing to the relevant section of the sample config for the full details.

This could also be done via our [upgrading docs](https://github.com/matrix-org/synapse/blob/develop/docs/upgrade.md), where we have more room.

Reply:
I would likely still sneak a keyword or two in there: `expiry_time` will be superseded by `expire_caches` and `cache_entry_ttl`.
Helps perk people's ears up and helps if you ever need to do a Ctrl+F for one of the config option names.